dura/trust_path/controller/logout.php

```diff
@@ -28,8 +28,17 @@ public function main()
 		$this->_default();
 	}
 
+	protected function _toRoom()
```

Comment:
Might be better to rename this as `_redirectToRoom()`, refer to lounge controller.